src/main/groovy/com/marklogic/gradle/task/schemas/LoadSchemasTask.groovy

```diff
@@ -0,0 +1,26 @@
+package com.marklogic.gradle.task.schemas
+
+import org.gradle.api.tasks.TaskAction
+
+import com.marklogic.appdeployer.AppDeployer
+import com.marklogic.appdeployer.command.schemas.LoadSchemasCommand
+import com.marklogic.appdeployer.impl.SimpleAppDeployer
+import com.marklogic.gradle.task.MarkLogicTask
+
+class LoadSchemasTask extends MarkLogicTask {
+
+    @TaskAction
+    void loadSchemas() {
+       LoadSchemasCommand command = null
+        // Check for a LoadSchemasCommand in the AppDeployer first, as that may have additional configuration that we
+        // don't want to have to duplicate on this task
+        AppDeployer d = getAppDeployer()
+        if (d instanceof SimpleAppDeployer) {
+            command = d.getCommand("mlLoadSchemasCommands")
```

Comment:
getCommand is a shortcut that allows you to fetch a command, from the list of commands in SimpleAppDeployer, based on its class name (not including its package). So the input here should be "LoadSchemasCommand".